main.js

```diff
@@ -192,4 +192,26 @@ btns.forEach((btn, index) => {
   btn.addEventListener('click', () => {
     popUp(index);
   });
-});
\ No newline at end of file
+});
+
+// eslint-disable-next-line array-bracket-spacing, comma-dangle
+const [form, ] = document.getElementsByClassName('contactform');
+const error = document.querySelector('.error');
```

Comment:
- [ optional] kindly make your error message color red so it can easily be noticed